translations/nl-be.json

```diff
@@ -998,7 +998,7 @@
   "date-approved": "Datum goedgekeurd",
   "consult-agendaitem-decision": "Laatste agendering bekijken",
   "postponed": "Uitgesteld",
-  "date-example": "bv. 01/2021",
+  "date-example": "bv. 01-2021",
```

Comment:
Deze lijkt mij niet juist aangezien de zoekfunctionaliteit niet aangepast is en er dus effectief `MM/YYYY` als input verwacht wordt in dat veld. Die zoekfunctionaliteit aanpassen indien een ander formaat gewenst zou zijn lijkt mij buiten scope van deze story en gaat ook conflicteren met #1288. Dus om deze PR te kunnen afronden zou ik deze change gewoon reverten.